tests/img_test.in

```diff
@@ -5,3 +5,4 @@ export MAIL_SCRIPT="$RUN_DIR/src/disrupt.sh"
 export FACTS_SOURCE="$TEST_DIR/img_test.lf"
 export IMAGE_RARITY=1
 export IMG_ASSET_DIR="${RUN_DIR}/tests/img_test_img"
+export LF_IMAGE_MAP="${RUN_DIR}/assets/lf_imap.json"
```

Comment:
Shouldn't this go under your ``imap_tests.in`` file?

Reply:
same as before, TODO for now